jtune.py

```diff
@@ -1753,6 +1756,41 @@ def _at_exit(raw_gc_log=None, jmap_data=None, jstat_data=None,
                   optimized_for_ygcs_rate)
 
 
+def _generate_rotated_gc_log_name(gc_log_file, log_number):
```

Comment:
The function name and it's arguments is longer than what the function does. I'd probably get rid of it, and do what it does w/o the function (if that makes any sense).